src/server/server.js

```diff
@@ -124,6 +126,28 @@ function processPluginHtml(html, pluginId) {
     });
 }
 
+function proxyXHR(request, response) {
+    var requestURL = url.parse(unescape(request.query.rurl));
+    
+    request.headers.host = requestURL.host;
+    // fixes encoding issue
+    delete request.headers["accept-encoding"];
+
+    var options = {
+        host: requestURL.host,
+        path: requestURL.path,
+        port: requestURL.port,
+        method: request.method,
+        headers: request.headers
+    };
+    
+    var proxyCallback = function(proxyReponse) {
+        proxyReponse.pipe(response);
+    };
+    
+    http.request(options, proxyCallback).end();
+}
+
```

Comment:
I agree. I'll change the implementation to load this lazily, only if the feature is enabled.